src/bundle/Resources/views/Security/login.html.twig

```diff
@@ -2,33 +2,37 @@
-                            </div>
-                            <div class="form-group{% if error %} has-error{% endif %}">
-                                <label for="password">{{ 'authentication.password'|trans|desc('Password') }}</label>
-                                <input type="password" id="password" class="form-control" name="_password" required="required" placeholder="{{ 'authentication.password.placeholder'|trans|desc('Enter password') }}" />
-                            </div>
+    <div class="row ez-login">
+        <div class="col-md-6 ez-login__image"></div>
+        <div class="col-md-6 ez-login__form">
+            <div class="ez-login__form-wrapper">
+                <svg class="ez-icon ez-icon--logo">
+                    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/bundles/ezplatformadminui/img/ez-logo.svg#logo-white"></use>
+                </svg>
+                <form action="{{ path( 'login_check' ) }}" method="post" role="form">
+                    <fieldset>
+                        <div class="form-group">
+                            <label class="ez-login__form__label" for="username">{{ 'authentication.username'|trans|desc('Username') }}:</label>
+                            <input type="text" id="username" class="ez-login__form__input form-control" name="_username" value="{{ last_username }}" required="required" autofocus="autofocus" autocomplete="on"/>
+                        </div>
+                        <div class="form-group{% if error %} has-error{% endif %}">
+                            <label class="ez-login__form__label" for="password">{{ 'authentication.password'|trans|desc('Password') }}:</label>
+                            <input type="password" id="password" class="ez-login__form__input form-control" name="_password" required="required"/>
+                        </div>
 
-                            <input type="hidden" name="_csrf_token" value="{{ csrf_token("authenticate") }}" />
-                            <input type="hidden" name="_target_path" value="{{ url('ezplatform.dashboard') }}" />
+                        <input type="hidden" name="_csrf_token" value="{{ csrf_token("authenticate") }}" />
+                        <input type="hidden" name="_target_path" value="{{ url('ezplatform.dashboard') }}" />
 
-                            <button type="submit" class="btn btn-primary">{{ 'authentication.submit'|trans|desc('Login') }}</button>
-                        </fieldset>
-                    </form>
-                </div>
+                        <button type="submit" class="btn btn-primary ez-btn ez-btn--login">{{ 'authentication.submit'|trans|desc('Login') }}</button>
+                    </fieldset>
+                </form>
+            </div>
+            <div class="ez-login__footer">
+                <span>eZ Platform ©1999-2017 <a href="http://ez.no">eZ Systems AS</a> and others.</span>
```

Comment:
Can we use `{{ "now"|date("Y") }}` instead of hardcoding current year ? 😉

Reply:
Yes, this is good idea!

Reply:
I didn't know how to do it in PHP. I will add it ;)